### PR TITLE
DO NOT PUBLISH. Peak demand performance

### PR DESCRIPTION
**Pull request by akristen**
New learning path about setting up and using New Relic to meet peak demand during an event~ WIP

### CODE REVIEW

File: src/content/docs/journey-peak-demand/choosing-relevant-service-levels.mdx [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+This tutorial explains why service levels are important, how to determine which capabilities need service levels, and how to create them for a gameday.
+
+## Why service levels? [#why]
+
+If you're used to thinking in terms of entities, identifying what transactions to prioritize can be tricky, and the whole point of capacity planning is prioritizing the right things. Even with existing service levels, it's good practice to reasses their effectivity and create new ones to fit new business needs. 
+
+Before getting started with creating service levels, we have some preliminary questions you might ask yourself about your architecture:
+
+* What journeys do your customers most frequently go through?
+* Of those, journeys involve purchase transactions?
```
Comment (akristen) [inline src/content/docs/journey-peak-demand/choosing-relevant-service-levels.mdx:19]: which* journeys

File: src/nav/journey-demand.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+      - title: Scale your infrastructure
+        path: /docs/journey-peak-demand/scale-infra
+      - title: Choose your service levels
+        path: /docs/journey-peak-demand/choosing-relevant-service-levels
+      - title: Create workloads
+        path: /docs/journey-peak-demand/create-dashboards-workloads
+      - title: Reduce alert noise
+        path: /docs/journey-peak-demand/create-quality-alerts
+      - title: Respond to incidents
+        path: /docs/journey-peak-demand/responding-to-incidents
```
Comment (tabathadelane) [inline src/nav/journey-demand.yml:15]:
```suggestion
  - title: Preparing for peak demand
    path:   /docs/journey-demand
    pages:
      - title: Get started
        path: /docs/journey-demand/succeed-with-new-relic
      - title: Scale your infrastructure
        path: /docs/journey-demand/scale-infra
      - title: Choose your service levels
        path: /docs/journey-demand/choosing-relevant-service-levels
      - title: Create workloads
        path: /docs/journey-demand/create-dashboards-workloads
      - title: Reduce alert noise
        path: /docs/journey-demand/create-quality-alerts
      - title: Respond to incidents
        path: /docs/journey-demand/responding-to-incidents
```

File: src/nav/root.yml [hunk 3]
```diff
@@ -10,6 +10,8 @@ pages:
     path: journey-performance
   - title: Migrate from Datadog 
     path: journey-migration
+  - title: Prepare for peak demand
+    path: journey-peak-demand
```
Comment (tabathadelane) [inline src/nav/root.yml:14]:
```suggestion
    path: journey-demand
```

File: src/nav/journey-demand.yml (hunk 2 above)
Comment (tabathadelane) [inline src/nav/journey-demand.yml:15]:
```suggestion
  - title: Preparing for peak demand
    path:   /docs/journey-demand
    pages:
      - title: Get started
        path: /docs/journey-demand/succeed-with-new-relic
      - title: Scale your infrastructure
        path: /docs/journey-demand/scale-infra
      - title: Choose your service levels
        path: /docs/journey-demand/choosing-relevant-service-levels
      - title: Create workloads
        path: /docs/journey-demand/create-dashboards-workloads
      - title: Reduce alert noise
        path: /docs/journey-demand/create-quality-alerts
      - title: Respond to incidents
        path: /docs/journey-demand/responding-to-incidents
```

**Pull request closed without merging**